docs/uploader/transloadit.mdx

```diff
@@ -287,17 +271,9 @@ pass user input from a `<form>` to a Transloadit Assembly:
 // This will add form field values to each file's `.meta` object:
 uppy.use(Form, { getMetaFromForm: true });
 uppy.use(Transloadit, {
-	getAssemblyOptions(file) {
-		return {
-			params: {
-				/* ... */
-			},
-			// Pass through the fields you need:
-			fields: {
-				message: file.meta.message,
-			},
-		};
-	},
+	assemblyOptions() {
+      // ...
```

Comment:
I think we should keep the explicit `return` statement, as otherwise it's relatively easy to miss the fact we're defining a method and not an object here.

Reply:
I think it distracts for the main point, which is how `Tranloadit` and `Form` work together. Plus the fact that the example above already shows what to put inside, so with `// ..` I'm referring back to that a bit.